src/ui.cpp

```diff
@@ -627,9 +633,8 @@ void uilist::calc_data()
     if( desc_enabled ) {
         desc_lines = 0;
         for( const uilist_entry &ent : entries ) {
-            // -2 for borders, -2 for padding
             desc_lines = std::max<int>( desc_lines, foldstring( footer_text.empty() ? ent.desc : footer_text,
-                                        w_width - 4 ).size() );
+                                        w_width - WIDTH_BUFFER ).size() );
```

Comment:
Not needed for description.

Reply:
resolved by https://github.com/CleverRaven/Cataclysm-DDA/pull/64478/commits/2fbc662ed6e32eee9b009acc12f23fd661beb2c0